attestation/file/file.go

```diff
@@ -30,11 +33,16 @@ import (
 // returned map of artifacts.
 func RecordArtifacts(basePath string, baseArtifacts map[string]cryptoutil.DigestSet, hashes []crypto.Hash, visitedSymlinks map[string]struct{}) (map[string]cryptoutil.DigestSet, error) {
 	artifacts := make(map[string]cryptoutil.DigestSet)
+	trail := omnitrail.New(omnitrail.WithSha1(), omnitrail.WithSha256())
```

Comment:
We should be able to pass in an exclude pattern, for example do we want to include the .git directory?

Reply:
That's a good idea, we could accept a regex pattern easily.

Reply:
we use a glob pattern in products:  https://github.com/testifysec/go-witness/blob/main/attestation/product/product.go#L49